Makefile

```diff
@@ -50,3 +51,9 @@ tarball: dist/$(PKGNAME)-$(PKGVER).tar.gz
 
 dist/$(PKGNAME)-$(PKGVER).tar.gz:
 	$(PYTHON) setup.py sdist
+
+%: %.in Makefile
```

Comment:
Is there any reason, why this rule is generic? There is no other file except `rhc-worker-playbook.spec` that could be generated using following command. I also believe that we want to make this target phony target, because you want to have unique NVR every time (the target should be always out-of-date). I suggest to change this line to something like this:

```Makefile
.PHONY: rhc-worker-playbook.spec
rhc-worker-playbook.spec: rhc-worker-playbook.spec.in
```

Reply:
The `rhc-worker-playbook.spec` is generated file now. This file should be removed, when running `make clean`.

Reply:
note that `.PHONY` is for marking targets that do not depend on any real file; if you need to force the regeneration of a target (no matter the dependencies), use `.FORCE`

Reply:
Making it a generic rule just makes things slightly more future-proof. In case
additional files need to be generated in the future, this sets that up. I don't
feel strongly one way or the other; if you'd prefer to restrict this rule to
just the spec file, I can do that too.